.github/workflows/merge_pull_request.yml

```diff
@@ -11,6 +11,17 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Resolve possible conflicts ${{ github.event.client_payload.ReleaseBranchName }} with main
```

Comment:
Why would there be conflicts? I rather not blindly do this, seems dangerous.

Reply:
Repro steps:
1) Create branch `test` from `main`, update readme in that branch, create PR to `main`
2) Update readme in main branch
3) PR can't be merged due to conflicts.

It is often case, when we still deploy `Week 1` on Ring 3 and starting to deploy `Week 2` to Ring 0.
Docs from Week 1 updates `main` branch on step 2.

Resolving conflicts this way looks safety because:
1) `PR branch` has only the single change in comparison with `main`: change in documentation file.
2) We trigger this workflow when we have finished deployment of particular week. It means that we 100% would like to take new documentation from our branch and replace any other docs in `main`. This process is pretty similar to what we had previously, when we just updated documentation file directly in master blindly

Reply:
Oh, discussed with @vsafonkin offline and realized issue with this approach. This approach pull `main` branch to release branch and this way we modify release branch and change state from what image was generated. We shouldn't do that

Reply:
So, I've reworked this approach. The special branch for pull request is creating as clone of release branch. It allows to resolve conflicts on PRs safely, without depends for release branch.
@maxim-lobanov @alepauly